src/Sarif/Core/SerializedPropertyInfoEqualityComparer.cs

```diff
@@ -24,17 +24,12 @@ public bool Equals(SerializedPropertyInfo left, SerializedPropertyInfo right)
                 return false;
             }
 
-            return left.SerializedValue.Equals(right.SerializedValue);
+            return left.Equals(right);
         }
 
         public int GetHashCode(SerializedPropertyInfo serializedPropertyInfo)
         {
-            if (serializedPropertyInfo.SerializedValue == null)
-            {
-                return 0;
-            }
-
```

Comment:
will the logic change after we remove this, do we still return same 0 when it is null

Reply:
The `SerializedPropertyInfoEqualityComparer.GetHashCode()` was not used, `serializedPropertyInfo.GetHashCode()` is the one been called. Here the change is to make SerializedPropertyInfoEqualityComparer to point to right code.